flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
in your description you say it's cross-platform. So we restricting to a single architecture?

Reply:
You're right. I've updated the description. For the record, the app is available in both arm64 and x86_64 versions, but I decided not to package the arm64 version for Flatpak.

Reply:
why not?

Reply:
It's my first time publishing something on Flathub, and I didn't want to mess up the manifest, so I played it safe. Besides, there aren't many ARM Linux devices, and we can still add the ARM version on Flatpak when a new release of Beaver comes out.

Reply:
Yeah I have heard that one a lot. And still never happen. So I will reiterate the request.